src/main/java/au/org/emii/ncdfgenerator/VariableEncoder.java

```diff
@@ -14,44 +14,43 @@
 import au.org.emii.ncdfgenerator.AttributeValue;
 
 
+
 class VariableEncoder implements IVariableEncoder
```

Comment:
I know this hasn't changed in this PR but VariableEncoder always thrown me off.  Would VariableDefinition be a better description?